.gitlab-ci.yml

```diff
@@ -341,3 +341,48 @@ submit_preflight_redhat_public_tag:
     - export RH_PARTNER_API_TOKEN=$(aws ssm get-parameter --region us-east-1 --name ci.datadog-operator.$RH_PARTNER_API_KEY_SSM_KEY --with-decryption --query "Parameter.Value" --out text)
     - export IMG=$RH_PARTNER_REGISTRY/$RH_PARTNER_PROJECT_ID:${CI_COMMIT_TAG:1}
     - make preflight-redhat-container-submit
+
+
+publish_community_operators:
+  stage: post-release
+  rules:
+      - if: $CI_COMMIT_TAG
+        when: manual
+      - when: never
+  needs:
+    - "submit_preflight_redhat_public_tag"
+  tags: [ "runner:docker", "size:large" ]
+  image: $JOB_DOCKER_IMAGE
+  before_script:
+    - mkdir -p .cache
+    - make install-tools
+  script:
+    # Set version
+    - VERSION=${CI_COMMIT_TAG:1} # vX.Y.Z => X.Y.Z
+    # Install github-cli and skopeo deps
+    - curl -fsSL "https://cli.github.com/packages/githubcli-archive-keyring.gpg" -o /usr/share/keyrings/githubcli-archive-keyring.gpg
+    - echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" > /etc/apt/sources.list.d/github-cli.list
+    - apt-get update -y
+    - apt-get install -y gh libgpgme-dev libassuan-dev libbtrfs-dev libdevmapper-dev pkg-config
+    - rm -rf /var/lib/apt/lists/*
+    # Build skopeo and install (needed for make bundle-redhat)
+    # Note: skopeo apt package is only available in for ubuntu 20.10+
+    # https://github.com/containers/skopeo/blob/main/install.md#ubuntu
+    # TODO: use apt-get when gitlab runner image base is updated to use ubuntu 20.10+
+    - git clone https://github.com/containers/skopeo $GOPATH/src/github.com/containers/skopeo
+    - cd $GOPATH/src/github.com/containers/skopeo && DISABLE_DOCS=1 make bin/skopeo
+    - DISABLE_DOCS=1 make install
+    # Set up redhat registry access to generate bundle-redhat
+    - mkdir ~/.redhat
+    - aws ssm get-parameter --region us-east-1 --name
+      ci.datadog-operator.redhat-registry-token --with-decryption --query "Parameter.Value" --out text > ~/.redhat/auths.json
+    - cd $CI_PROJECT_DIR
+    - make bundle-redhat
+    # configure git and github-cli
+    - git config --global user.email $GITLAB_USER_EMAIL
+    - git config --global user.name $GITLAB_USER_NAME
+    - aws ssm get-parameter --region us-east-1 --name ci.datadog-operator.github-robot-token --with-decryption --query "Parameter.Value" --out text > git-token.txt
+    - gh auth login --with-token < git-token.txt
+    - gh auth setup-git
+    # create pull request for each marketplace repo
+    - $CI_PROJECT_DIR/.gitlab/community_operators_prs.sh $VERSION community-operators community-operators-prod certified-operators redhat-marketplace-operators
```

Comment:
nit: maybe we could add a `make` command and move the script to `hack/` (even though it's not a hack script, that's where most of our scripts live atm)